### DISCUSSION TITLE
No Webserver

### DISCUSSION THREAD

**Discussion by KinkyOne** [pdulvp/jellyfin-qnap#42] [General]
Hi Fokls,
iam new to Jellyfin and just installed them on my TS-431K.
Installation without any problems, but no webpage on port 8092.

had a emby-server running on my machine (also listens on 8092), but uninstalled before jelly inst.

any ideas ??

**Comment by Neokeys** [comment]
I am also having problems getting Jellyfin to connect to plugins and I asume this is a connection probelm?
Port 8092 is open on my router so would appreciate any advice.
Oh and it's not a certificate issue as my Nas has a valid certificate installed from Let's Encrypt.
Thank you in advance.
